### PR TITLE
add ticket 6 GET

### PR DESCRIPTION
**Pull request by Anna702**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -112,3 +112,41 @@ describe("/api/articles/:article_id", () => {
...
+        expect(Array.isArray(body)).toBe(true);
+        expect(body.length).toBe(2);
+        body.forEach((comment) => {
+          expect(comment).toHaveProperty("comment_id", expect.any(Number));
+          expect(comment).toHaveProperty("author", expect.any(String));
+          expect(comment).toHaveProperty("article_id", 9);
+          expect(comment).toHaveProperty("body", expect.any(String));
+          expect(comment).toHaveProperty("created_at", expect.any(String));
+          expect(comment).toHaveProperty("votes", expect.any(Number));
+        });
```
Comment (pippaamy) [inline __tests__/app.test.js:131]: make sure the comments are sorted by created_at

File: __tests__/app.test.js [hunk 2]
```diff
@@ -112,3 +112,41 @@ describe("/api/articles/:article_id", () => {
...
+      });
```
Comment (pippaamy) [inline __tests__/app.test.js:132]: make a 200 test to test when it has a valid but has no comments responds with an empty array of comments

**Review by pippaamy — CHANGES_REQUESTED** [review]
Good work Anna, a few tests to add! Can you add these and then put in a new pull request?

**Review by JVal-NC — COMMENTED** [review]
Good work Anna, go ahead and merge

**Pull request merged**